app/controllers/works_controller.rb

```diff
@@ -91,4 +97,14 @@ def category_from_work
     render_404 unless @work
     @media_category = @work.category.downcase.pluralize
   end
+
+  def require_user
+    current_user = User.find_by(id: session[:user_id])
+    if current_user
+      true
```

Comment:
I love that you made this a controller filter. However, you've got the same helper method in both the works and users controllers - could you put this in the application controller to help DRY it up further?